compute_tools/src/extension_server.rs

```diff
@@ -123,16 +123,41 @@ pub async fn get_available_extensions(
         .download_stream
         .read_to_end(&mut ext_idx_buffer)
         .await?;
-    let ext_index_str = str::from_utf8(&ext_idx_buffer).expect("error parsing json");
+    let ext_index_str = str::from_utf8(&ext_idx_buffer).expect("json err");
```

Comment:
`std::str::from_utf8` is not needed, `serde_json::from_slice` works in place of `serde_json::from_str`. (UTF-8 is only possible for json.)